apps/framework-cli/src/cli/routines/scripts.rs

```diff
@@ -111,10 +111,18 @@ pub async fn run_workflow(
         .start(&project.temporal_config, input)
         .await
         .map_err(|e| {
+            // Remove trailing ', details: [...]' and ', metadata: ...' from error string if present
```

Comment:
Do we still need this? I suspect it's already trimmed down through `concise_msg` below